### ISSUE TITLE
forge.min.js causes parse error with gulp-ng-annotate

### ISSUE THREAD

**Issue by henry74** [digitalbazaar/forge#228]
I have gulp-ng-annotating all my js files and it runs into the following error with forge.min.js:
```
scripts/app-66d41dd1.js: error: couldn't process source due to parse error
Line 332: Delete of an unqualified identifier in strict mode.
```
From stackoverflow: http://stackoverflow.com/questions/16652589/why-is-delete-not-allowed-in-javascript5-strict-mode

The line number is likely to be incorrect.  I have many js files included, but this is the only one which runs into this error.  I've tried using bower to install forge, but it does not run correctly with the bower install.  None of the objects are available and it's not clear how to instantiate them using the bower install.

**Comment by davidlehn** [comment]
You shouldn't need to run forge through that tool since it doesn't do angular DI but I'll guess it's just easier that way with your build setup? I tried the minimal config from the `gulp-ng-annotate` README and it processed `forge.min.js` and output the same js as the input, as expected. I'll take a guess that the problem is that you are processing forge along with some other code that sets global strict mode. I added `'use strict';` to the top of `forge.min.js` and see the same issue you see. It's likely due to some of the module boilerplate forge uses. The easiest solution is to not process forge along with your other code. Another solution is to use an IIFE in your code so strict mode doesn't apply globally. If you put `'use strict';` at the top of a file and the concatenate files, the strict mode will apply to all the code, even included libraries that do not work in strict mode.  Try surrounding your code with something like this:

```js
(function() {
  'use strict';
  ... your code ...
})();
```
